microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -481,7 +485,6 @@ Your browsing history and other data are transferred over HTTPS and attached to
 
 You may modify your interests or opt-out of personalized ads from the [Ad settings](https://account.microsoft.com/privacy/ad-settings) on the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/).
 
-> [!NOTE]
 > Opting out of personalized ads on the [Microsoft privacy dashboard](https://account.microsoft.com/privacy/) does not turn off the collection and use of your browsing history for personalization of search results and content in your news feed. You can turn off the collection and use of your Microsoft Edge browsing history for personalized search results and news. Go to `edge://settings/privacy`. In **Personalize your web experience** turn off the **Improve your web experience by allowing Microsoft to use your browsing history from the account for personalizing advertising, search, news and other Microsoft services** setting. If you stop sharing the data, Microsoft no longer collects and uses your browsing history to personalize ads, search results, and news. For more information about personalization in Microsoft Edge, see [Microsoft Edge browsing history for personalized advertising and experiences](https://support.microsoft.com/help/4532583).
```

Comment:
The tagging is probably broken since Note tagging line is gone.  Such Alerts (eg Note colored rectangle) are deprecated.  Alternatives are:
```
**Note: ** Foo.
```

```
#### Make a label for the note as an h4 heading
```
DCG:   https://learn.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning - "Avoid Notes, Tips, and Important boxes. Readers tend to skip over them. It's better to put that info directly into the article text.

If you need to use alerts, limit them to one or two per article. Multiple notes should never be next to each other in an article."